src/node/transaction.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2017-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_NODE_TRANSACTION_H
+#define BITCOIN_NODE_TRANSACTION_H
+
+#include <primitives/transaction.h>
+#include <uint256.h>
+
+typedef enum TransactionError_t {
```

Comment:
This looks like a C-ism. You can just use `enum TransactionError { ... };` in C++ with (I think) the same effect.

Reply:
In my defense I copied it from ScriptError_t. I'll change it.